lib/svg-sprite/mode/standalone.js

```diff
@@ -89,13 +89,20 @@ SVGSpriteStandalone.prototype._layout = function(files, cb, extend) {
                 }
             });
 
+            // Create the SVG setter
             Object.defineProperty(this.data.shapes[index], '_svg', {
                 enumerable: false,
                 writable: true
             });
 
-            this.data.shapes[index].__defineSetter__('svg', function(svg) {
-                this._svg = svg;
+            // Create the SVG getter/setter
+            Object.defineProperty(this.data.shapes[index], 'svg', {
+                get() {
```

Comment:
This might be redundant

Reply:
Maybe that was a part of the plan to have only setters here? I'm a little confused

Reply:
I was getting an ESLint error (https://eslint.org/docs/rules/accessor-pairs) but happy to disable this one case if we are sure we only need a setter.

Feel free to push to this branch.

Reply:
It's just more likely that originally it just was forgotten. So in this case it is ok.